server/core/src/logic/transaction_operations.rs

```diff
@@ -44,6 +44,18 @@ pub async fn create_transaction(
     Ok(transaction.id)
 }
 
+pub async fn find_transactions_by_expense_id(
+    conn: &DatabaseConnection,
+    expense_id: Id,
+) -> Result<Vec<transaction::Model>, DbErr> {
+    let transactions = Transaction::find()
```

Comment:
Maybe the `find_related` function would be better for this. More [here](https://www.sea-ql.org/SeaORM/docs/basic-crud/select/#find-related-models).

Reply:
Hey Mister, using `find_with_related` now. Also modified your test according to my changes.
See my comment for the test below.